_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -19,15 +19,400 @@ Subnets can be now associated with IPAM plugins providing IP address management
 capabilities via external entities. An experimental implementaion for phpIPAM
 is available as a [Smart Proxy plugin](https://github.com/grizzthedj/smart_proxy_ipam).
 
-### Release Notes
+#### Rails 6 upgrade
+
+Foreman has been updated to run on the latest version of Ruby on Rails at the time of the release, 6.0.2. Many plugins have already been updated as well, but there may be issues with any that have not seen a new release in a while.
+
+#### Support installation on EL8 distributions
+
+Foreman installation is now possible on EL8 distributions. The installer has been updated to support changes needed for these and yum repositories now contain packages for EL8. Note that installation is only supported and tested on RHEL and CentOS, packages may or may not work on Oracle Linux or Scientific Linux.
```

Comment:
This probably requires some changes in other parts of the manual, but i'm not sure what exactly needs to be changed.

Reply:
We've also discussed the quality level and not everyone was comfortable with production quality. Given we've made changes right up before branching, it hasn't seen a lot of testing. One suggestion was to label it beta quality, maybe even for 2.1.0 GA.

Reply:
Can you please add that in a follow up PR as well as the needed changes in the installation instructions? I wasn't very involved in the el8 work so i'm not sure what changes are needed.